addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1240,6 +1240,54 @@ registry.ImagesLazyLoading = publicWidget.Widget.extend({
     },
 });
 
+registry.BackgroundShape = publicWidget.Widget.extend({
```

Comment:
@qsm-odoo Not sure if it is the right place to add this code.. As all the shape stuff are in web editor...
But since the fix is for website pages, I figured it wasn't so bad to do it here. What do you think ?

Reply:
It's fine in website